src/libraries/Common/tests/System/Security/Cryptography/MLKemTests.Keys.cs

```diff
@@ -0,0 +1,698 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Looking at dir structure I think this should be under `AlgorithmImplementations` dir

Reply:
Yeah it could be, though to be honest I don't see why we have the structure that way. I have been trying to simplify things over the years (trying to get rid of `Internal.Cryptography` and trying to reduce the amount of deep directory nesting.

If we feel strongly about continuing to follow this directory structure I can move it.

Reply:
I think that's a legacy from pre-1.0, when RSA and DSA weren't even in the same assembly.  (They got merged into Algorithms during 1.0).

Some directories are good for reducing clutter.. but since the src project is just grouped by namespace, and all the stuff lives in S.S.C, I'm supportive of "let's try something new" with these.